rubrics/tool/pom.xml

```diff
@@ -198,6 +198,16 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.itextpdf</groupId>
+            <artifactId>itextpdf</artifactId>
+            <version>5.5.13.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jsoup</groupId>
```

Comment:
Do not force the version here, bind it to the master version here:

https://github.com/sakaiproject/sakai/blob/master/master/pom.xml#L48

This means removing the version.

Reply:
Also itext dependency? @mpellicer

Reply:
Nop, only JSOUP: